internal/daemon/device-plugin/deviceplugin.go

```diff
@@ -229,17 +230,17 @@ func (dp *dpServer) connectWithRetry(endpoint string) (*grpc.ClientConn, error)
 	return conn, nil
 }
 
-// func (dp *dpServer) Stop() error {
-// 	fmt.Printf("Stopping Device Plugin gRPC server..")
-// 	if dp.grpcServer == nil {
-// 		return nil
-// 	}
+func (dp *dpServer) Stop() error {
+	dp.log.Info("Stopping Device Plugin...")
+	if dp.grpcServer == nil {
+		return nil
+	}
 
-// 	dp.grpcServer.Stop()
-// 	dp.grpcServer = nil
+	dp.grpcServer.Stop()
```

Comment:
This isn't correct. We need to refactor and pull out `dp.grpcServer.Serve` from `RegisterDevicePlugin`, and properly implement stopping.

Reply:
I implemented the proper stopping. PTAL